blockonomics.php

```diff
@@ -33,7 +33,7 @@ public function __construct()
     {
         $this->name = 'blockonomics';
         $this->tab = 'payments_gateways';
-        $this->version = '1.8.0';
+        $this->version = '1.7.91';
```

Comment:
@shivaenigma, I [recommend](https://github.com/blockonomics/prestashop-plugin/pull/106#discussion_r619146236 ) using the version numbers v1.7.9.1, v1.7.9.2 etc until Prestashop 1.8 is released in order to avoid repeating https://github.com/blockonomics/prestashop-plugin/issues/101 in the future, and avoid limiting our future releases for PS 1.7 to v1.7.9.91->v1.7.9.99

Reply:
Problem is prestashop validator doesn't allow 1.x.x.x format. We incremented versions too fast!

Reply:
I tested, and it's possible to upgrade from **1.7.9** -> **1.7.10**. That way we still have 90 increments, which should be sufficient.

Reply:
WE already have had 1.7.83 release so we cannot go lower https://github.com/blockonomics/prestashop-plugin/tags

Lets use 1.7.91

Reply:
Done, changed the version to 1.7.91.